From a0f85aba06479bec8f14080417bd0a2b48997624 Mon Sep 17 00:00:00 2001
From: Yehuda Yitschak <yehuday@marvell.com>
Date: Mon, 14 Mar 2016 17:15:12 +0200
Subject: [PATCH 0252/2241] config: added io virtualization features

macvtap/lan - to enable virtio networking support
vhost - to accelerate virtio networking
vfio - to enable device assignment to guest
openvswitch - for optimal communication between guests

Change-Id: I26a11bba4040567b36fe44501a4bd0b88d2197c7
Signed-off-by: Yehuda Yitschak <yehuday@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/28226
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 arch/arm64/configs/mvebu_v8_lsp_defconfig | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/arm64/configs/mvebu_v8_lsp_defconfig b/arch/arm64/configs/mvebu_v8_lsp_defconfig
index 19c7cfb..034acca 100644
--- a/arch/arm64/configs/mvebu_v8_lsp_defconfig
+++ b/arch/arm64/configs/mvebu_v8_lsp_defconfig
@@ -58,6 +58,7 @@ CONFIG_IP_PNP_BOOTP=y
 # CONFIG_IPV6 is not set
 CONFIG_BRIDGE=y
 CONFIG_VLAN_8021Q=y
+CONFIG_OPENVSWITCH=y
 CONFIG_BPF_JIT=y
 # CONFIG_WIRELESS is not set
 CONFIG_NET_9P=y
@@ -97,8 +98,11 @@ CONFIG_MD_RAID1=y
 CONFIG_MD_RAID10=y
 CONFIG_MD_RAID456=y
 CONFIG_NETDEVICES=y
+CONFIG_MACVLAN=y
+CONFIG_MACVTAP=y
 CONFIG_TUN=y
 CONFIG_VIRTIO_NET=y
+CONFIG_VHOST_NET=y
 CONFIG_E1000=y
 CONFIG_E1000E=y
 CONFIG_IGB=y
@@ -178,9 +182,14 @@ CONFIG_DMADEVICES=y
 CONFIG_MV_XOR=y
 CONFIG_MV_XOR_V2=y
 CONFIG_DMATEST=y
+CONFIG_VFIO=y
+CONFIG_VFIO_PCI=y
+CONFIG_VFIO_PLATFORM=y
+CONFIG_VIRT_DRIVERS=y
 CONFIG_VIRTIO_PCI=y
 CONFIG_VIRTIO_BALLOON=y
 CONFIG_VIRTIO_MMIO=y
+CONFIG_VIRTIO_MMIO_CMDLINE_DEVICES=y
 CONFIG_COMMON_CLK_VERSATILE=y
 CONFIG_CLK_SP810=y
 CONFIG_CLK_VEXPRESS_OSC=y
@@ -230,3 +239,4 @@ CONFIG_CRYPTO_AES_ARM64_CE_CCM=y
 CONFIG_CRYPTO_AES_ARM64_CE_BLK=y
 CONFIG_CRYPTO_AES_ARM64_NEON_BLK=y
 CONFIG_CRYPTO_CRC32_ARM64=y
+CONFIG_CRC_CCITT=y
-- 
2.7.4

